src/packages/utils/src/things/json-rpc/json-rpc-quantity.ts

```diff
@@ -1,79 +1,103 @@
-import { bufferToBigInt, BUFFER_EMPTY } from "../../utils";
-import { BaseJsonRpcType } from "./json-rpc-base-types";
-// TODO(perf): rewrite this stuff since it isn't really caching anything
+import { bufferToBigInt } from "../../utils/buffer-to-bigint";
+import { BaseJsonRpcType, JsonRpcInputArg } from "./json-rpc-base-types";
+const BUFFER_EMPTY = Buffer.alloc(0);
```

Comment:
Do we want to take this from `constants.ts`?

Reply:
I did, but then we get circular references :/

I thought about removing the static RPC-JSON types from constants, and moving them onto the classes themselves - ie
```
class Quantity ... {

Reply:
I've moved them to statics as a separate commit so that we can revert easily if desired - keen to hear your thoughts on this approach: https://github.com/trufflesuite/ganache/pull/2983/commits/93f6d565a7e617d5fbbaaaa331beb8ef24d82ea7

Reply:
I like it!